File: composeApp/src/androidMain/kotlin/com/hossian/noteappkmp/presentation/note_list/NoteListState.kt
package com.hossian.noteappkmp.presentation.note_list

import com.hossian.noteappkmp.domain.model.Note
import com.hossian.noteappkmp.utils.NoteOrder
import com.hossian.noteappkmp.utils.OrderType

data class NoteListState(
    val notes: List<Note> = emptyList(),
    val noteOrder: NoteOrder = NoteOrder.Date(OrderType.Descending),
    val isOrderSectionVisible : Boolean = false
)

File: iosApp/iosApp/presentation/note_list/NoteListState.swift
//
//  NoteListState.swift
//  iosApp
//
//  Created by Hossain Muktar on 17/9/24.
//  Copyright © 2024 orgName. All rights reserved.
//

import Foundation
import Shared

struct NoteListState {
    var notes: [Note] = []
    var noteOrder: NoteOrder = NoteOrder.Date(orderType: OrderType.Descending())
    var isOrderSectionVisible: Bool = false
}
